src/main/java/it/gov/pagopa/payhub/activities/activity/classifications/ClearClassifyIuvActivity.java

```diff
@@ -0,0 +1,20 @@
+package it.gov.pagopa.payhub.activities.activity.classifications;
+
+import io.temporal.activity.ActivityInterface;
+import io.temporal.activity.ActivityMethod;
+
+/**
+ * Interface for defining an activity to delete classifications based on IUV.
+ */
+@ActivityInterface
+public interface ClearClassifyIuvActivity {
+	/**
+	 * deletion of a classification based on the provided parameters
+	 *
+	 * @param organizationId organization id
+	 * @param iuv creditor reference identifier
+	 * @return boolean true if success deletion or false if fails
+	 */
+	@ActivityMethod
+	boolean deleteClassificationByIuv(Long organizationId, String iuv);
```

Comment:
@marcomatteuccieng is it right to classify all the entities related to the IUV (all the related transfer) at the same time?
If yes, we could let all the workflows which involve IUV, to reason just on the IUV (forgetting about the transfers): the IUV classification will classify all the installments (an installment is entirely paid, it cannot be paid partially)